tests/lib/rules/filename-blocklist.posix.js

```diff
@@ -236,3 +236,24 @@ ruleTester.run('filename-blocklist with option: []', rule, {
     },
   ],
 });
+
+ruleTester.run(
+  "filename-blocklist with option: [{'*.models.ts': 'Some Non Glob reason'}, true]",
+  rule,
+  {
+    valid: [
+      {
+        code: "var foo = 'bar';",
+        filename: 'src/foo.apis.ts',
+        options: [
+          {
+            '*.models.ts': 'Some Non Glob reason',
+          },
+          true,
+        ],
+      },
+    ],
+
+    invalid: [],
+  }
+);
```

Comment:
Would it be possible to add another invalid test case and add the same test cases to the windows style test file `filename-blocklist.windows.js`?